playwright/BootTests/Content/ContentNonRepeatable.boot.ts

```diff
@@ -0,0 +1,389 @@
+/* eslint-disable playwright/no-wait-for-timeout */
+import { expect } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+
+import { test } from '../../fixtures/customizations';
+import { isHosted } from '../../helpers/helpers';
+import { ensureAuthenticated } from '../../helpers/login';
+import {
+  fillInImageOutput,
+  ibFrame,
+  navigateToLandingPage,
+} from '../../helpers/navHelpers';
+import {
+  createBlueprint,
+  deleteBlueprint,
+  fillInDetails,
+  registerLater,
+} from '../../helpers/wizardHelpers';
+import { deleteRepository, navigateToRepositories } from '../helpers/helpers';
+import {
+  buildImage,
+  constructFilePath,
+  downloadImage,
+} from '../helpers/imageBuilding';
+import { OpenStackWrapper } from '../helpers/OpenStackWrapper';
+
+test('Content integration test - Non repeatable build - URL source', async ({
+  page,
+  cleanup,
+}) => {
+  test.skip(
+    !isHosted(),
+    'Skipping test. Boot test run only on the hosted service.',
+  );
+  const blueprintName = 'content-non-repeatable-test-' + uuidv4();
+  const filePath = constructFilePath(blueprintName, 'qcow2');
+  const repositoryName = 'content-non-repeatable-test-' + uuidv4().slice(0, 8);
+  const repositoryUrl =
+    'https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/';
+  const packageName = 'cockateel';
+
+  // Here we want to be sure that the repository is deleted due to URL exclusivity per repository
+  await deleteRepository(page, repositoryUrl);
+
+  // Delete the blueprint compliance policy and Openstack resources after the run
```

Comment:
I assume this was just a leftover from copying base :nerd_face: 

```suggestion
  // Delete the blueprint, repository and Openstack resources after the run
```